src/test/java/org/broadinstitute/dsde/consent/ontology/integration/Term.java

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.dsde.consent.ontology.integration;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Term {
```

Comment:
Actually, while I really like this idea, it will take a bit of work to merge/validate that nothing else breaks.